Sources/Datadog/DDRUMMonitor.swift

```diff
@@ -286,6 +286,10 @@ public class DDRUMMonitor {
     /// - Parameter key: key for the attribute that will be removed.
     public func removeAttribute(forKey key: AttributeKey) {}
 
+    public var _internal: _RUMInternalProxy? {
+        get { return nil }
+    }
```

Comment:
Here, this could be:
```suggestion
    public internal(set) var _internal: _RUMInternalProxy? = nil
```
Then, you don't need to override it in `RUMMonitor`

Reply:
Ah yeah, okay -- I think you left the initial  comment on the wrong variable. I think initially I wanted _internal to be non-optional which is why I used the getter, but that didn't work well. I'll modify it to do this instead.